app/Config/App.php

```diff
@@ -310,40 +319,11 @@ class App extends BaseConfig
 	 * will be set on cookies. If set to `None`, `$cookieSecure` must also be set.
 	 *
 	 * @var string
+	 * 
+	 * @deprecated use Config\Cookie::$samesite property instead.
 	 */
 	public $cookieSameSite = 'Lax';
 
-	/**
-	 * --------------------------------------------------------------------------
-	 * Cookie Raw
-	 * --------------------------------------------------------------------------
-	 *
-	 * This flag allows setting a "raw" cookie, i.e., its name and value are
-	 * not URL encoded using `rawurlencode()`.
-	 *
-	 * If this is set to `true`, cookie names should be compliant of RFC 2616's
-	 * list of allowed characters.
-	 *
-	 * @var boolean
-	 *
-	 * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#attributes
-	 * @see https://tools.ietf.org/html/rfc2616#section-2.2
-	 */
-	public $cookieRaw = false;
```

Comment:
this is BC break, existing user no longer can't use `(new Config\App())->cookieRaw;`

Reply:
(new Config\App())->cookieRaw; wasn't released yet,

Reply:
oh, ok 👍